Documentation and main script improved (#1)
edvgui/tst-utils: README.md

# TST Utils

## Introduction

This project contains some helpers to parse a trade republic tst file (pdf report) and use the output data to fill in the belgian TST file that should be sent to state.

This code-base is purely opportunistic, and will rely on the simplest tools it can find to reach its goal.  Consistency of the technology used is not the primary goal, having a solution that works is.

## Prerequisites

The requirements are: 
```
- python>=3.10
- podman 
```
Some tweaking with the gmail api is also necessary, see [tst-sender-readme](./tst-sender/README.md).

## Input files 

The script requires various input files in order to generate a correct TST pdf.

> The complete list with is the following:

Reviewer: ```suggestion
The complete list is the following:
```

> - Personal file informations as json file with the following format:

Reviewer: ```suggestion
- Personal information file, as json payload, with the following format:
```

```
{
    "fullName": "FIRSTNAME LASTNAME",
    "nationalRegisterNumber": "",
    "address": [""]
}
```

- Signature file as png or jpg, in order to sign the TST file.

- The credentials json file used to access your gmail account, more information available [here](./tst-sender/README.md).

## Quick start

To run the bash script, just run:

```
sh sync.sh
```

The options are defined as follow:

```

> [xxxxxx@xxxxxx tst-utils-riushda]$ sh sync.sh -h

Reviewer: ```suggestion
[xxxxxx@xxxxxx tst-utils]$ sh sync.sh -h
```

Tool to fill in belgian TST file from trade republic monthly report.
This tool also creates a draft mail with the generated document and a qr code to pay the tax.

Syntax: sh sync.sh [-h|s|p|c] input_folder output_folder
options:
h     Display this help and exit.
s     Set the signature image file path, default value is 'data/signature.jpg'.

> p     Set the personal informations json file path, default value is 'data/citizen.json'.

Reviewer: ```suggestion
p     Set the personal information json file path, default value is 'data/citizen.json'.
```
